packages/node_modules/@ciscospark/react-component-presence-avatar/src/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import styles from './styles.css';
+
+import Avatar from '@ciscospark/react-component-avatar';
+import Icon, {ICON_TYPE_DND, ICON_TYPE_PTO} from '@ciscospark/react-component-icon';
+
+function PresenceAvatar(props) {
+  const {
+    isSelfAvatar,
+    presenceStatus
+  } = props;
+
+  let icon;
+  const avatarClasses = [`ciscospark-presence-avatar`, styles.avatar];
+  // Do not show presence on self avatars
+  if (!isSelfAvatar) {
+    if (presenceStatus === `active`) {
+      avatarClasses.push(`ciscospark-presence-avatar-active`);
+      avatarClasses.push(styles.active);
+    }
+    else if (presenceStatus === `inactive`) {
+      avatarClasses.push(`ciscospark-presence-avatar-inactive`);
+      avatarClasses.push(styles.inactive);
+    }
+    else if (presenceStatus === `ooo`) {
+      avatarClasses.push(`ciscospark-presence-avatar-ooo`);
+      avatarClasses.push(styles.ooo);
+      icon = <Icon type={ICON_TYPE_PTO} />;
+    }
+    else if (presenceStatus === `dnd`) {
+      avatarClasses.push(`ciscospark-presence-avatar-dnd`);
+      avatarClasses.push(styles.dnd);
+      icon = <Icon type={ICON_TYPE_DND} />;
+    }
+  }
+
+  return (
+    <div className={classNames(avatarClasses)}>
+      {icon &&
+        <div className={classNames(`ciscospark-presence-avatar-icon-container`, styles.iconContainer)} >
+          <div
+            className={classNames(`ciscospark-presence-avatar-icon`, styles.icon)}
+            style={{fontSize: props.iconSize}}
```

Comment:
Should we keep the same setup as we did in the Avatar component and have size be: `small`, `medium`, `large`? I definitely prefer that to allowing arbitrary avatar sizes. Will make it easier later to unilaterally change sizes when the new Spring designs come in.

Reply:
I think we've removed every instance of the named sizes from all avatar implementations in favor of arbitrary sizes everywhere.